opcua/events/streams.go

```diff
@@ -108,8 +109,12 @@ func (es *eventHandler) Handle(ctx context.Context, event events.Event) error {
 
 func decodeCreateThing(event map[string]interface{}) (createThingEvent, error) {
 	strmeta := read(event, "metadata", "{}")
+	meta, err := base64.StdEncoding.DecodeString(strmeta)
```

Comment:
events are not base64 encoded why are we decoding from base64?

Reply:
Metadata is base64 encoded since it is marshaled as a slice of bytes

Reply:
![image](https://github.com/absmach/magistrala/assets/43080232/fc5a34e9-308a-4dee-9b87-169b3d77b7bd)